supervision/tracker/byte_tracker/core.py

```diff
@@ -67,9 +67,11 @@ def activate(self, kalman_filter, frame_id):
         self.state = TrackState.Tracked
         if frame_id == 1:
             self.is_activated = True
+            if self.minimum_consecutive_frames == 1:
+                self.external_track_id = self.next_external_id()
 
-        if self.minimum_consecutive_frames == 1:
-            self.external_track_id = self.next_external_id()
+        # if self.minimum_consecutive_frames == 1:
```

Comment:
Do we aim to keep that comment here?

Reply:
I don't think we actually need it. I will remove it.